src/navigation/CustomDrawerContentComponent.js

```diff
@@ -20,8 +20,8 @@ export const CustomDrawerContentComponent = ({ navigation, drawerRoutes, state }
 
   return (
     <DiagonalGradient>
-      <SafeAreaView>
-        <View style={stylesWithProps({ orientation }).header}>
+      <SafeAreaView style={styles.safeAreaView}>
+        <View style={styles.header}>
```

Comment:
right that the style is not depending on orientation. but that change could be reverted to have less code changes?
on the other hand it makes sense to separate the styles depending on orientation and not depending. hm

Reply:
i reverted the style changes for now.